src/charm.py

```diff
@@ -207,7 +272,12 @@ def ready_to_configure(self) -> bool:
         if self._get_invalid_configs():
             return False
 
-        for relation in ["fiveg_nrf", "database", "certificates"]:
+        for relation in [
+            FIVEG_NRF_RELATION_NAME,
+            "database",
+            TLS_RELATION_NAME,
+            SDCORE_CONFIG_RELATION_NAME
+        ]:
```

Comment:
If we extract this to a separate method, we won't have to ignore the complexity error.

Reply:
You are right. We generally followed an approach to put all the checks in a single method which causes complexity errors. This situation also exists in other charms.

Reply:
Done.